app/src/main/java/org/jellyfin/androidtv/util/BlurHashDecoder.kt

```diff
@@ -0,0 +1,197 @@
+package org.jellyfin.androidtv.util
+
```

Comment:
Can you add a link to the original source (include the commit hash in it please)